__tests__/Fieldset-test.tsx

```diff
@@ -1,7 +1,9 @@
+/// <reference path="../typings/globals/mocha/index.d.ts" />
```

Comment:
What made this necessary?

Reply:
Also, with moving the tests, I would probably match the directory structure under src, and we can ditch the underscores (I think maybe that was a React convention for indicating it wasn't part of the source?). So I'd go with `tests/components/Fieldset-test.tsx`.

Reply:
Sounds good. It was necessary because of the upgrade in `ts-lint`. I'm not exactly sure what it was but the config and how it builds is different.

Reply:
`tsc` does create a `__tests__` folder for you so that may be the convention. To be consistent, I'm leaving the main test folder as `__tests__` too, but I moved the tests to `__tests_/components/....tsx`.

Reply:
I don't like this reference - it's a global dependency, so it needs to be loaded for all the tests and just happens to be in this one because it's the first test file. That means it could easily break if you add or remove tests or rename files. I think it would be better to put it in the compiler options in the tsconfig file, but I'm having trouble getting it to work and I think it may be a bug with Typescript 2.4.1. If you're planning to upgrade Typescript farther than that in a future PR, we could revisit this then.